File: app/src/main/java/com/cere/csplayer/until/FileUtils.java
package com.cere.csplayer.until;

import java.io.File;

/**
 * Created by CheRevir on 2019/10/24
 */
public class FileUtils {
    public static boolean isExists(String path) {
        return new File(path).exists();
    }

    public static void delete(String path) {
        File file = new File(path);
        if (file.exists()) {
            file.delete();
        }
    }
}

File: app/src/main/java/com/cere/csplayer/util/FileUtils.kt
package com.cere.csplayer.util

import java.io.File

/**
 * Created by CheRevir on 2019/10/24
 */
object FileUtils {

    @JvmStatic
    fun isExists(path: String): Boolean {
        return File(path).exists()
    }

    @JvmStatic
    fun delete(path: String) {
        val file = File(path)
        if (file.exists()) {
            file.delete()
        }
    }

}